com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -1533,7 +1536,12 @@ TextureHandle DoTemporalAntialiasing(RenderGraph renderGraph, HDCamera hdCamera,
                         }
 
                         ctx.cmd.DrawProcedural(Matrix4x4.identity, data.temporalAAMaterial, 0, MeshTopology.Triangles, 3, 1, mpb);
-                        ctx.cmd.DrawProcedural(Matrix4x4.identity, data.temporalAAMaterial, 1, MeshTopology.Triangles, 3, 1, mpb);
+#if UNITY_EDITOR
+                        if (ShaderUtil.IsPassCompiled(data.temporalAAMaterial, 1))
```

Comment:
note: instead of this we could add #pragma editor_sync_compilation in the TAA shader, guess it would be cleaner